docs/reference/module-options.md

```diff
@@ -811,3 +843,198 @@ module.exports = {
+
+**It is important that the widget type has a [configured `widgetEditor` component](#components) that is built for this purpose.** If there is no such component, the widget editor modal will open immediately on load.
+
+#### Example
+
+```javascript
+// modules/@apostrophecms/layout-widget/index.js
+module.exports = {
+  options: {
+    contextual: true,
+    components: {
+      // 👇 This refers to a project-level `MyCustomLayoutWidgetEditor.vue`
+      // component file.
+      widgetEditor: 'MyCustomLayoutWidgetEditor',
+      widget: 'AposWidget'
+    }
+  },
+  // ...
+}
+```
+-->
+
+### `icon`
+
+Identify an icon to be used with a widget label in the area menu with the `icon` option. That icon must be included in the [list of globally available UI icons](https://github.com/apostrophecms/apostrophe/blob/3.0/modules/@apostrophecms/asset/lib/globalIcons.js) or configured on the module in its `icons` section. See the [module format example](/guide/module-format-example.md) for how to make new icons available.
+<!-- TODO: Update this to link to a true module section documentation page for `icons`. -->
+
+#### Example
+
+```javascript
+// modules/two-column-widget/index.js
+module.exports = {
+  extend: '@apostrophecms/widget-type',
+  options: {
+    icon: 'pillar'
+  },
+  icons: {
+    pillar: 'Pillar'
+  },
+  // ...
+};
+
+```
+
+!['Area menu with icons next to widget labels'](/images/area-menu-with-icons.png)
+
+### `label` (for widgets)
+
+`label` should be set to a text string to be used in the area menu. If not set, Apostrophe will convert the module `name` meta property to a readable label by removing `-widget` from the end, splitting the `name` on dashes and underscores, then capitalizing the first letter of each word.
+
+#### Example
+
+```javascript
+// modules/two-column-widget/index.js
+module.exports = {
+  extend: '@apostrophecms/widget-type',
+  options: {
+    label: 'Two Column Layout'
+  },
+  // ...
+};
+
+```
+
+<!-- TODO: Flesh out once questions around scenes are resolve, or delete. -->
+<!-- ### `scene` -->
+
+## Options for the core rich text widget
+
+Option settings in this section apply to the core rich text widget module (`@apostrophecms/rich-text-widget`).
+
+| Option | Value type | Description |
+|---------|---------|---------|
+| [`defaultData`](#defaultdata) | Object | Define initial default data for rich text content. |
+| [`defaultOptions`](#defaultoptions) | Object | Configure the rich text toolbar and styles for rich text widgets. |
+| [`editorTools`](#editortools) | Object | Configure rich text tools and their Vue components. |
+
+### `defaultData`
+
+Rich text widgets can start with default content by setting `defaultData` to an object with a `content` property. That value would be a string of text or HTML that all rich text widgets would include when added.
```

Comment:
I didn't know we had this! Is this like, placeholder data? more or less?

Reply:
Not exactly. Or sort of? If you leave it, it'll get saved and be public. So not placeholder in the sense that it's only visible to editors (like text input placeholders).

Reply:
Interesting. I love this though, we could definitely use it in Demos I think.